src/components/StyleGuide/Filtering/PageCompositionElements/FilterBar.mdx

```diff
@@ -0,0 +1,232 @@
+import { Meta } from "@storybook/blocks";
+
+import { Box } from "@chakra-ui/react";
+import Banner from "../../../Banner/Banner";
+import Image from "../../../Image/Image";
+import Link from "../../../Link/Link";
+
+import imageExampleColumn from "/static/filteringStyleGuide/exampleColumn.png";
+import imageExamplePopup from "/static/filteringStyleGuide/examplePopup.png";
+import imageExampleRow from "/static/filteringStyleGuide/exampleRow.png";
+import imageFilterBar from "/static/filteringStyleGuide/filterBar.png";
+import { exampleWrapperStyles } from "../Overview.mdx";
+
+<Meta title="Style Guide/Filtering/Page elements/Filter bar" />
+
+# Filter bar
+
+The `filter bar` is a wrapper for individual form input elements, including the
+following DS components: `CheckboxGroup`, `MultiSelectGroup`, `RadioGroup`,
+`Checkbox`, `DatePicker`, `MultiSelect`, `Radio`, `Select`, `Slider`,
+`TextInput`, and `Toggle`.
+
+<Image
+  alt="Example of filter bar"
+  src={imageFilterBar}
+  {...exampleWrapperStyles}
+/>
+
+## Table of contents
+
+- {<Link href="#rendering" target="_self">Rendering</Link>}
+- {<Link href="#filter-option-labeling" target="_self">Filter option labeling</Link>}
+- {<Link href="#filter-option-status" target="_self">Filter option status</Link>}
+- {<Link href="#action-buttons" target="_self">Action buttons</Link>}
+
+## Rendering
+
+The `filter bar` element should be rendered using the DS
+[FilterBarInline](./?path=/docs/components-form-elements-filterbarinline--docs)
+and
+[FilterBarPopup](./?path=/docs/components-form-elements-filterbarpopup--docs)
+components.
+
+<Banner
+  content={
+    <>
+      <strong>IMPORTANT:</strong> The filter bar should be configured based on
```

Comment:
Yes, but note, RC only uses `FilterBarInline` for both mobile and desktop.

Reply:
Yes, true. But these are recommendations for a more ideal state. This is what apps should be working toward.